packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -147,9 +180,69 @@ export const prepareLocalchainAccountKit = (baggage, makeRecorderKit, zcf) => {
         makeTransferAccountInvitation() {
           throw Error('not yet implemented');
         },
+        /** @type {LocalChainAccount['deposit']} */
+        async deposit(payment) {
+          return E(this.facets.helper.owned()).deposit(payment);
+        },
+        /** @type {LocalChainAccount['withdraw']} */
+        async withdraw(amount) {
+          return E(this.facets.helper.owned()).withdraw(amount);
+        },
+        /**
+         * @returns {ChainAddress['address']}
+         */
+        getAddress() {
+          return NonNullish(this.state.address, 'Chain address not available.');
+        },
+        /**
+         * @param {AmountArg} amount an ERTP {@link Amount} or a {@link DenomAmount}
+         * @param {ChainAddress} destination
+         * @param {IBCMsgTransferOptions} [opts] if either timeoutHeight or timeoutTimestamp are not supplied, a default timeoutTimestamp will be set for 5 minutes in the future
+         * @returns {Promise<void>}
+         */
+        async transfer(amount, destination, opts) {
+          trace('Transferring funds from LCA over IBC');
+          // TODO #9211 lookup denom from brand
+          !matches(amount, AmountShape) || Fail`ERTP Amounts not yet supported`;
+
+          // TODO #8879 chainInfo and #9063 well-known chains
+          const { transferChannel } = agoricChainInfo.connections.get(
+            destination.chainId,
+          );
+
+          await null;
+          // set a `timeoutTimestamp` if caller does not supply either `timeoutHeight` or `timeoutTimestamp`
+          // TODO #9324 what's a reasonable default? currently 5 minutes
+          const timeoutTimestamp =
+            opts?.timeoutTimestamp ??
+            (opts?.timeoutHeight
+              ? 0n
+              : await timestampHelper.getTimeoutTimestampNS());
+
+          const [result] = await E(this.facets.helper.owned()).executeTx([
+            typedJson('/ibc.applications.transfer.v1.MsgTransfer', {
+              sourcePort: transferChannel.portId,
+              sourceChannel: transferChannel.channelId,
+              token: {
+                amount: String(amount.value),
+                // @ts-expect-error AmountArg already narrowed to DenomAmount
```

Comment:
if you change the `!matches(...) || Fail...` to `if (typeof amount !== 'string') throw ...`, I think you won't need this override.

Reply:
`if ('brand' in amount)` makes it happy. `in`, more precisely

Reply:
I frequently forget to do this instead of `if (amount.brand)`. Hopefully after writing it out now, I won't 🙂